### ISSUE TITLE
Change polling time?

### ISSUE THREAD

**Issue by leconnaisseur** [And3rsL/Deebot-for-Hassio#69]
Hey And3rsL,

first of all: THANKS for the plugin! works like a charm on my Home Assistant! I'm running it with a modified vacuum card on lovelace and created automations for it to clean specific rooms at specific times and days. Really great implementation, so thanks for that!
My only question: Is it possible to change the polling time? My firewall and pihole are lighting up with requests to portal-eu.ecouser.net every 20 seconds. Do you think it's possible to change that to something in the region of every 1-5 minutes while still maintaining scheduled cleaning triggers? Or would that break the entire thing?

Thanks again and have a great 2021!

**Comment by philosowaffle** [comment]
+1 for this request.

According to AdGuard, in the past 24 hours my homeassistant raspberry pi issued 400,000 requests to `portal-na.ecouser.net`.  Its been doing this for a while, would be nice to decrease the polling time.

**Comment by skynetua** [comment]
Noticed that as well, it is probably `deebotozmo` but there are very often calls to api server(almost every second) in HA core logs, like: 
```
2021-02-09 11:38:26 DEBUG (schedule_thread_statuses) [deebotozmo.ecovacsjson] Calling IOT api with {'auth': {'realm': 'ecouser.net', 'resource':   .... }
2021-02-09 11:38:26 DEBUG (schedule_thread_statuses) [deebotozmo.ecovacsjson] Got { ... }
```

**Comment by And3rsL** [comment]
> Noticed that as well, it is probably `deebotozmo` lib issue but there are very often calls to api server(almost every second) in HA core logs, like:
> 
> ```
> 2021-02-09 11:38:26 DEBUG (schedule_thread_statuses) [deebotozmo.ecovacsjson] Calling IOT api with {'auth': {'realm': 'ecouser.net', 'resource':   .... }
> 2021-02-09 11:38:26 DEBUG (schedule_thread_statuses) [deebotozmo.ecovacsjson] Got { ... }
> ```

The lib is triggered by the custom component, im working for differential refresh timers. stay tuned for next updates

**Comment by leconnaisseur** [comment]
> 
> 
> > Noticed that as well, it is probably `deebotozmo` lib issue but there are very often calls to api server(almost every second) in HA core logs, like:
> > ```
> > 2021-02-09 11:38:26 DEBUG (schedule_thread_statuses) [deebotozmo.ecovacsjson] Calling IOT api with {'auth': {'realm': 'ecouser.net', 'resource':   .... }
> > 2021-02-09 11:38:26 DEBUG (schedule_thread_statuses) [deebotozmo.ecovacsjson] Got { ... }
> > ```
> 
> The lib is triggered by the custom component, im working for different refresh timers. stay tuned for next updates

Thanks for the update!

**Comment by adampetrovic** [comment]
FWIW i'm seeing about 6 DNS requests a second to `portal-ww.ecouser.net`

![image](https://user-images.githubusercontent.com/88030/107731992-59b5e600-6d4b-11eb-8800-ca2bdd19209a.png)

**Comment by And3rsL** [comment]
> FWIW i'm seeing about 6 DNS requests a second to `portal-ww.ecouser.net`
> 
> ![image](https://user-images.githubusercontent.com/88030/107731992-59b5e600-6d4b-11eb-8800-ca2bdd19209a.png)
> 

Do you have live map enabled ?

**Comment by adampetrovic** [comment]
Yes I do.

**Comment by And3rsL** [comment]
> Yes I do.
> 

Then is ok, however it shouldn't update live map when is not cleaning

Ecovacs server API are not well designed, to get a map i should do +20 API request

**Comment by sefki21** [comment]
Hi @And3rsL 

Not sure but as this is closed, I assume it was fixed in 2.1.1? 

I had iRobots in the past and am new to Ecovacs (with my T9), map via HA is turned off but I have about 14‘000 polls to portal-na.ecouser.net in 24h. 

Do I need to adjust something?
